karabo_bridge/client.py

```diff
@@ -40,6 +41,8 @@ class Client:
     ser : str, optional
         Serialization protocol to use to decode the incoming message (default
         is msgpack) - supported: msgpack,pickle.
+    timeout : int
+        Timeout on :method:`next` (in milliseconds)
```

Comment:
Do we have any timeouts elsewhere? For Jupyter we've tried to keep all timeout parameters in (fractional) seconds so you don't have to look up the units every time you specify a timeout. But it's the consistency that's useful, so if we use milliseconds elsewhere, :+1:

Reply:
All in favour of sticking to SI units (i.e. seconds) where we can.

Reply:
changed it to seconds